assets/js/playlist.js

```diff
@@ -36,4 +35,17 @@
           $('#validation_error').append(error.responseJSON.link);
       }
     });
-  });
\ No newline at end of file
+  });
+
+  // ajax for deleting songs
+  $(document).on('click', '.deleteSong', function( event ){
+    $.ajax({
+      method: 'GET',
```

Comment:
delete function should be used POST method instead of GET

Reply:
Updated.